acceptance/framework/environment/environment.go

```diff
@@ -21,15 +21,17 @@ import (
 )
 
 const (
-	DefaultContextName   = "default"
-	SecondaryContextName = "secondary"
+	DefaultContextName         = "default"
+	SecondaryContextNamePrefix = "secondary"
```

Comment:
I think we should probably remove the secondary context const and the specific functions. Now that we have more than one, it'd be nice to change it to:
- get the default (same as today)
- get specific context by index. Ideally the default one should just call the specific one with index `0`

Reply:
I'll create a separate commit for this refactor and you can let me know if that's what you had in mind